perf_test/sparse/KokkosSparse_spmv_struct_tuning.cpp

```diff
@@ -101,7 +143,7 @@ void struct_matvec(const int stencil_type,
                    const AMatrix& A,
                    const XVector& x,
                    typename YVector::const_value_type& beta,
-                   const YVector& y,
+                   YVector& y,
```

Comment:
Why not const?  It's a Kokkos::View.

Reply:
I could bring back the const, the intent is to show that this view could be modified, although its data is really what is being modified.